turbinia/e2e/e2e-local.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# This scripts executes a Turbinia end-to-end test against a local 
+# docker-compose Turbinia stack
+# The evidence processed is Chrome Browser History.
+
+echo "Create evidence folder"
+mkdir -p ./evidence
+sudo chmod 777 ./evidence
+
+echo "Download Chrome browser history artifact"
+curl https://raw.githubusercontent.com/obsidianforensics/hindsight/master/tests/fixtures/profiles/60/History > History
+tar -vzcf ./evidence/history.tgz History
+ls -al ./evidence/
+
+echo "Startup local turbinia docker-compose stack"
+docker-compose -f ./docker/local/docker-compose.yml up -d
+
+echo "Sleep for 5s"
+sleep 5s
+
+echo "Show running instances"
+docker ps -a
+
+echo "Show container logs"
+docker logs turbinia-server
+docker logs turbinia-worker
+
+echo "Create  Turbinia request"
+docker exec -t turbinia-server turbiniactl -r 123456789 compresseddirectory -l /evidence/history.tgz 
+
+echo "Sleep for 60 seconds to let Turbinia process evidence"
+sleep 60s
+
+echo "Display Turbinia request status"
+docker exec turbinia-server turbiniactl -a status -r 123456789
```

Comment:
Do we want to look at the output of this to verify that the run was successful here?  Also, to save some execution time, we could potentially remove the sleep above, and call it with `turbiniactl -w` to wait for the request to finish.

Reply:
We can do that after we fix below issue.
https://github.com/google/turbinia/issues/749

I think I would like to iteratively improve the e2e test with new evidence to process and a mechanism to decide on what is a successful e2e test and what is not.

Reply:
Added a check to see if any task failed as a minimal exit level check.